### DISCUSSION TITLE
is there any NFT marketplace that runs on devnet?

### DISCUSSION THREAD

**Discussion by sweetim** [aptos-labs/aptos-developer-discussions#139] [Questions] [answered]
i am writing NFT contract, and would like to test my NFT if it is configured correctly, is there something similiar like `OpenSEA` in devnet?
i have tried all the NFT listed here (https://aptosfoundation.org/ecosystem/projects/nft-tooling), but none of them support devnet?

**Comment by 0x-j** [comment, marked as answer]
Hi, we have an example marketplace deployed on devnet, [link](https://nft-playground.gnazar.io/?network=devnet)

[Frontend code](https://github.com/gregnazario/aptos-test-nft-mint)

[Contract code](https://github.com/aptos-labs/aptos-core/tree/main/aptos-move/move-examples/marketplace)

**Reply by gregnazario** [reply]
This example marketplace does not get redeployed on every devnet reset.  It is live on testnet though
